lib/backend/k8s/k8s.go

```diff
@@ -263,20 +250,24 @@ func (c *KubeClient) EnsureInitialized() error {
 }
 
 func (c *KubeClient) Clean() error {
```

Comment:
Could you add a comment (now or later) to say this method is just for the tests?